internal/services/machinelearning/machine_learning_compute_cluster_resource.go

```diff
@@ -61,54 +61,57 @@ func resourceComputeCluster() *pluginsdk.Resource {
 				ValidateFunc: validation.StringInSlice([]string{string(machinelearningservices.VMPriorityDedicated), string(machinelearningservices.VMPriorityLowPriority)}, false),
 			},
 
-			"identity": {
+			"identity": SystemAssignedUserAssigned{}.Schema(),
+
+			"scale_settings": {
 				Type:     pluginsdk.TypeList,
 				Required: true,
 				ForceNew: true,
 				MaxItems: 1,
 				Elem: &pluginsdk.Resource{
 					Schema: map[string]*pluginsdk.Schema{
-						"type": {
-							Type:     pluginsdk.TypeString,
+						"max_node_count": {
+							Type:     pluginsdk.TypeInt,
 							Required: true,
 							ForceNew: true,
-							ValidateFunc: validation.StringInSlice([]string{
-								string(machinelearningservices.ResourceIdentityTypeSystemAssigned),
-							}, false),
 						},
-						"principal_id": {
-							Type:     pluginsdk.TypeString,
-							Computed: true,
+						"min_node_count": {
+							Type:     pluginsdk.TypeInt,
+							Required: true,
+							ForceNew: true,
 						},
-						"tenant_id": {
+						"scale_down_nodes_after_idle_duration": {
 							Type:     pluginsdk.TypeString,
-							Computed: true,
+							Required: true,
+							ForceNew: true,
 						},
 					},
 				},
 			},
 
-			"scale_settings": {
+			"ssh_settings": {
```

Comment:
settings is redundant here
```suggestion
			"ssh": {
```